packages/coinstac-client-core/src/index.js

```diff
@@ -225,10 +225,11 @@ class CoinstacClient {
    * @returns {Promise}
    */
   _initAuthorization(credentials) {
-    if (credentials.email && credentials.name) {
-      return this.auth.createUser(credentials);
-    }
-    return this.auth.login(credentials);
+    const doLogin = () => this.auth.login(credentials);
+
+    return credentials.email && credentials.name ?
+      this.auth.createUser(credentials).then(doLogin) :
```

Comment:
Big change in _coinstac-client-core_ is here: create that user, then do that login.